src/Error/BaseErrorHandler.php

```diff
@@ -148,9 +148,19 @@ public function handleError($code, $description, $file = null, $line = null, $co
 
         $debug = Configure::read('debug');
         if ($debug) {
+            // By default trim 3 frames off for the public and protected methods
+            // used by ErrorHandler instances.
+            $start = 3;
+
+            // Can be used by error handlers that wrap other error handlers
+            // to coerce the generated stack trace to the correct point.
+            if (isset($context['_trace_offset'])) {
```

Comment:
I would probably make this name more direct.  `_trace_frame_offset` and change `start` to `start_frame`.

It could then be described as an offset from the starting frame which makes sense with the math here.

Reply:
Changing `start` would be a breaking change as it modifies the API of `Debugger::outputError()`.

Reply:
I meant to say would be nice to change `start`. I still think making this name mention `frame` is helpful for when random cake devs debug in the future.